### PR TITLE
Fix images in README.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Update image links.

### CODE REVIEW

**Comment by AnotherButler** [comment]
@tubaman

File: README.md [hunk 1]
```diff
@@ -24,17 +24,17 @@ For our in-house production, we have used manual sanding and painting for the be
 
 #### Rendered images
 
-![](images/earwig-01.png)
-![](images/earwig-02.png)
-![](images/earwig-03.png)
+<span class="images">![](https://s3.us-west-2.amazonaws.com/reference-docs-images/work_environ_mon_docs/earwig-01.png)</span>
```
Comment (tubaman) [inline README.md]: I'm getting a 403 for that s3 link:
wem_hardware_image_issue_403

**Review by tubaman — CHANGES_REQUESTED** [review]
Looks like I'm getting a 403 for those s3 links.  Can you check to see if the web team set permissions correctly?

**Review by tubaman — APPROVED** [review]
Looks like Phillip Howell has uploaded these to S3 for us!

**Pull request merged**